src/test/java/hw3/ex1/LoginPOTestWithSoftAssert.java

```diff
@@ -0,0 +1,72 @@
+package hw3.ex1;
+
+
+import hw3.LoginBasePO;
+import hw3.pages.IndexPage;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import java.util.Arrays;
+import java.util.List;
+
+
+public class LoginPOTestWithSoftAssert extends LoginBasePO {
+
+    @Test
+    public void loginTestWithSA() {
+        SoftAssert softAssert = new SoftAssert();
+        IndexPage indexPage = new IndexPage(getDriver());
+        //1. Open test site by URL
+        openSite();
+
+        //2. Assert Browser title
+        softAssert.assertEquals(indexPage.getBrowserTitle(), "Home Page", "Browser title has incorrect name");
+
+        //3. Perform login
+        indexPage.login("Roman", "Jdi1234");
```

Comment:
Why do you do not implement properties file? Please read requrements carefully

Reply:
I created PropertyReader class